.github/workflows/ci.yaml

```
@@ -22,3 +22,21 @@ jobs:
       run: make generate manifests && git diff --exit-code
     - name: Test
       run: make test
+    - name: Get version from tags
+      id: version
+      run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}
+    - name: Publish
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        KO_DOCKER_REPO: ghcr.io/${{ github.repository_owner }}
+        AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        REPOSITORY_OWNER: ${{ github.repository_owner }}
+        VERSION: ${{ steps.version.outputs.tag }}
+      run: |
+        echo "${AUTH_TOKEN}" | docker login ghcr.io --username ${REPOSITORY_OWNER} --password-stdin
+        go run -modfile hack/kustomize/go.mod sigs.k8s.io/kustomize/kustomize/v3 build config/default | go run -modfile hack/ko/go.mod github.com/google/ko resolve -B --tags ${VERSION} -f - > release.yaml
```

attempting to improve readability, your call to take or leave it

```suggestion
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        KO: go run -modfile hack/ko/go.mod github.com/google/ko
        KO_DOCKER_REPO: ghcr.io/${{ github.repository }}
        KUSTOMIZE: go run -modfile hack/kustomize/go.mod sigs.k8s.io/kustomize/kustomize/v3
        VERSION: ${{ steps.version.outputs.tag }}
      run: |
        echo "${GITHUB_TOKEN}" | docker login ghcr.io --username ${{ github.repository_owner }} --password-stdin
        ${KUSTOMIZE} build config/default | ${KO} resolve --tags "${VERSION}" -f - > release.yaml
```